source/code/utilities/chinese/chinese_translations.hcp

```diff
@@ -513,6 +533,16 @@ class
         entries.emplace_back(x);
     }
     
+    {
+        Translation_Entry x;
+        x.english = L"tree";
+        x.pinyin = L"shù";
+        x.chinese = L"樹";
+        x.tags.emplace_back("word");
+        x.tags.emplace_back("outside");
+        entries.emplace_back(x);
```

Comment:
these are great because one of the tips I got was to walk around and talk about what I'm seeing.  so I'm definitely going to make a "things I see on the way to work" deck.